### PR TITLE
feat: blockscout support with sc verification

### PR DESCRIPTION
**Pull request by franjoespejo**
addresses #372

### CODE REVIEW

**Comment by franjoespejo** [comment]
@barnabasbusa pls note:
- blockscout execution block explorer not activated by default
- up to date with latest changes in main
- resources limit values are just a guess
- not sure if its ok to return the 'blockscout verification url' from the main function so can be used in the diva-package that calls this eth-package

**Comment by barnabasbusa** [comment]
Could you add blockscout to a couple of the tests ? 

- `.github/tests/mix-with-tools.yaml`
- `.github/tests/mix-with-tools-mev.yaml`

Do you see a value in enabling persistent storage for this tool (if users asks for persistence) ?

**Comment by franjoespejo** [comment]
right, will do test and persistence!

**Comment by franjoespejo** [comment]
I also returned the url for the verification sc service, pls check, I can rm if needed

File: src/blockscout/blockscout_launcher.star [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+}
+
+
+def launch_blockscout(plan, el_client_contexts, persistent):
+    postgres_output = postgres.run(
+        plan,
+        service_name="{}-postgres".format(SERVICE_NAME_BLOCKSCOUT),
+        database="blockscout",
+        extra_configs=["max_connections=1000"],
+        persistent=persistent,
```
Comment (barnabasbusa) [inline src/blockscout/blockscout_launcher.star:47]: This won't enable persistence the way you expect it. 

It needs to be defined like this: 

https://github.com/kurtosis-tech/ethereum-package/blob/main/src/el/geth/geth_launcher.star#L333

So its the files parameter that you want to enable persistence on. And mount the pvc to whatever datadir you want to keep persistence of.

**Comment by barnabasbusa** [comment]
This probably has more to do with the image than with kurtosis config, so lets merge this.

**Review by barnabasbusa — APPROVED** [review]

**Pull request merged**